tests/viet_kwik_e_mart/app_async.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+"""This module contains the Kwik-E-Mart MindMeld demo application implemented in Vietnamese"""
+from mindmeld import Application
+
+
+app = Application(__name__, async_mode=True, language="vi")
+
+
+@app.handle(intent="greet")
+async def welcome(request, responder):
+    try:
+        responder.slots["name"] = request.context["name"]
+        prefix = "Chào {name}. "
+    except KeyError:
+        prefix = "Xin chào. "
+    responder.reply(
+        prefix + "Tôi có thể giúp bạn tìm giờ cửa hàng "
+        "cho Kwik-E-Mart tại địa phương của bạn. "
+        "Tôi có thể giúp gì cho bạn?"
```

Comment:
I like that pattern better in https://github.com/cisco/mindmeld/pull/111/files#diff-7ee0332813a566ef0aff8bd1968e56c2R17
where prefix is sent as a separate reply.

Reply:
So something like:

`responder.reply(prefix)`
`responder.reply(sentence1)`
`responder.reply(sentence2)`
`responder.reply(sentence3)`

@serapio So splitting into separate replies?